doc/game.md

```diff
@@ -150,6 +150,19 @@ built upon two methods:
 
 The `GameLoop` is used by all of Flame's `Game` implementations.
 
+# Pause/Resuming game execution
```

Comment:
We didn't had any docs bout this, Added this small mention about it here.

Reply:
These should be ##, but it seems to be consistently wrong in this file